src/QMCHamiltonians/HamiltonianFactory.cpp

```diff
@@ -176,8 +177,18 @@ bool HamiltonianFactory::build(xmlNodePtr cur)
         targetH->addOperator(std::move(hs), "Grid", true);
       }
     }
+    else if (cname == "flux")
+    {
+      if (potName.size() < 1)
+	potName = "flux";
+      targetH->addOperator(std::make_unique<ConservedEnergy>(), potName, false);
+    }
     else if (cname == "estimator")
```

Comment:
Change here like `cname == "estimator" || cname == "legacy_estimator"` and the error check as
`if (batched && cname == "estimator")`

Reply:
I disagree with this change, legacy estimators that are going to continue to be allowed should be explicitly changed as the flux estimator has been.  They are not "estimators" by the definition we will be using coming forward.  They should be renabled individually if necessary. Further discussion of this should return to #4637 which this PR addresses, if this is the functionality you think is correct argue for it in #4637 with @jtkrogel  and amend this PR or make another one.

Reply:
Changing to `flux` or `legacy_estimator`, I prefer the latter a more generic way.
We can only allow using `legacy_estimator` when batched driver is in use and the selected estimator doesn't have an actual batched implementation.